### ISSUE TITLE
BS4 (beautifulsoup4) dependency is missing the new "soupsieve" sub-dependency

### ISSUE THREAD

**Issue by polyglot-jones** [wbond/packagecontrol.io#166]
I have a package that specifies "bs4" (beautiful soup html scraper) in `dependencies.json`. My package worked until a month ago. Today, it fails with "Cannot execute CSS selectors because the soupsieve package is not installed." When I do Package Control: Satisfy Libraries, I get "Package Control: Installed library "beautifulsoup4" 4.12.3 for Python 3.8" which is the latest and greatest version of beautifulsoup4, which apparently now requires a new library called  soupsieve. 

The problem is that "https://packagecontrol.github.io/channel/channel_v4.json" (as of just now, 4/3/2024) refers to "https://files.pythonhosted.org/packages/b1/fe/e8c672695b37eecc5cbf43e1d0638d88d66ba3a44c4d321c796f4e59167f/beautifulsoup4-4.12.3-py3-none-any.whl", but it makes no mention to the now-required soupsieve wheel.

**Comment by FichteFoll** [comment]
Indeed, the latest version of bs4 (and most likely earlier ones as well) [requires soupsieve](https://git.launchpad.net/beautifulsoup/tree/pyproject.toml?h=4.12.3#n33), which is currently not available as a dependency for PC. I've [submitted a PR](https://github.com/packagecontrol/channel/pull/6) for it. Once it is merged, you should be able to also add `soupsieve` to your requirements (for the 3.8 host, i.e. ST4, only).
